identities/models.py

```diff
@@ -73,6 +73,26 @@ def serialize_hook(self, hook):
     def __str__(self):
         return str(self.id)
 
+    def save(self, *args, **kwargs):
+        if self.id and not self._state.adding:
```

Comment:
I see that it is only when user asks to be completely forgotten that the keys would change, it also changes the other fields to "redacted", I will use that to exclude them.